src/pudl/metadata/labels.py

```diff
@@ -37,6 +37,274 @@
     'F': 'Less than or equal to 5 minutes',
     'O': 'Other',
 }
+"""Descriptive labels for EIA momentary interruption codes."""
+
+POWER_PURCHASE_TYPES_FERC1: Dict[str, str] = {
+    'RQ': 'requirement',
+    'LF': 'long_firm',
+    'IF': 'intermediate_firm',
+    'SF': 'short_firm',
+    'LU': 'long_unit',
+    'IU': 'intermediate_unit',
+    'EX': 'electricity_exchange',
+    'OS': 'other_service',
+    'AD': 'adjustment'
+}
+"""Descriptive labels for FERC 1 power purchase type codes."""
+
+CENSUS_REGIONS: Dict[str, str] = {
+    'NEW': 'New England',
+    'MAT': 'Middle Atlantic',
+    'SAT': 'South Atlantic',
+    'ESC': 'East South Central',
+    'WSC': 'West South Central',
+    'ENC': 'East North Central',
+    'WNC': 'West North Central',
+    'MTN': 'Mountain',
+    'PACC': 'Pacific Contiguous (OR, WA, CA)',
+    'PACN': 'Pacific Non-Contiguous (AK, HI)',
+}
+"""Descriptive labels for Census Region codes."""
+
+RTO_ISO: Dict[str, str] = {
+    'CAISO': 'California ISO',
+    'ERCOT': 'Electric Reliability Council of Texas',
+    'ISONE': 'ISO New England',
+    'MISO': 'Midcontinent ISO',
+    'NYISO': 'New York ISO',
+    'PJM': 'PJM Interconnection',
+    'SPP': 'Southwest Power Pool'
+}
+"""Descriptive labels for RTO/ISO short codes."""
+
+NAICS_SECTOR_CONSOLIDATED_EIA: Dict[str, str] = {
+    # traditional regulated electric utilities
+    '1': 'Electric Utility',
+
```

Comment:
I was doing this to try and make it more clear which entry the comments pertain to.

Really though, it would be better if the longer, more legible description were actually available as data.